terraform/modules/happy-stack-eks/README.md

```diff
@@ -1,4 +1,4 @@
-<!-- bump2 -->
+<!-- bump3 -->
```

Comment:
What in the world is this?

Reply:
The datadog change is to a submodule which doesn't trigger the release of a parent module. This html comment change is enough to trigger the release of a parent module.